site/public/css/electronic.css

```diff
@@ -590,6 +590,22 @@ progress::-webkit-progress-value {
     border: 5px solid var(--standard-vibrant-pink-red);
 }
 
+/* stylelint-disable-next-line selector-id-pattern */
+#autograding_results .box-block .red-message {
+    background-color: #940000;
```

Comment:
Please use color from color.css to maintain uniform color scheme across entire system

Reply:
Oh okay. I will. Thank you for the guidance. Do you think "--alert-danger-red" will look good on this?

![image](https://github.com/Submitty/Submitty/assets/68075099/f5dc640b-ce4c-42b3-bc26-3d97ca73f265)

![image](https://github.com/Submitty/Submitty/assets/68075099/adffcaa3-bead-4522-8ece-95d91ca71f70)